addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientDetailsEdit.xml

```diff
@@ -105,7 +105,7 @@
                             <t t-foreach="env.pos.pricelists" t-as="pricelist"
                                t-key="pricelist.id">
                                 <option t-att-value="pricelist.id"
-                                        t-att-selected="props.partner.property_product_pricelist ? (pricelist.id === props.partner.property_product_pricelist[0] ? true : false) : false">
+                                        t-att-selected="props.partner.property_product_pricelist ? (pricelist.id === props.partner.property_product_pricelist[0] ? true : undefined) : pricelist.id === env.pos.default_pricelist.id ? true : undefined">
```

Comment:
Do we really need this?

Reply:
The issue is that when the default pricelist for the PoS is not the selected option, any other option can be selected and we can't be sure that if it's the correct one.
The alternative is to read the option when creating the contact, but it is more complex. I think having the default pricelist shown to the user is also more consistent, though I might be wrong.